src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -77,7 +77,7 @@ def _check_model_args(
     """
     Check if model name is provided.
     """
-    if not args.subcommand and not args.model:
+    if not args.model:
```

Comment:
should we check if the subcommand is profile? The model should be required for profile but not for compare.

Reply:
nvm, this will only run when the subcommand is profile 👍